### DISCUSSION TITLE
Error auto rotate - button slot

### DISCUSSION THREAD

**Discussion by bamiendev** [google/model-viewer#4628] [Q&A]
Hi, I'm having trouble with the auto-rotate function.
Specifically, I have a button as follows:
<button type="button" class="view-button" style="display: block;" slot="hotspot-0" 
data-position="3.2213281435636m 0.21651540409254721m 0.8202013642723696m" 
data-normal="0.3069375160393453m -0.1753928663081973m 0.9354286203103857m" 
data-orbit="30.6deg 76.9deg 13.4m" 
data-field-of-view="13.71deg" 
data-target="3.2647987982849935m 0.20708035569254576m 0.7982761468630228m " 
data-visibility-attribute="visible" data-visible=""> Nhóm 1</button>

I use Camera Views in Annotations so that when you press the button, it zooms closer to the set position, but after turning on auto-rot, that position is changed unnecessarily.
How to keep the assigned data-orbit and data-target intact when turning on the auto-rotate function?

**Comment by bamiendev** [comment]
![1](https://github.com/google/model-viewer/assets/116520283/b21d35a5-ef15-4f9d-820e-79a123f0ee28)

**Reply by elalish** [reply]
I'm not quite sure what you mean. When I turn on auto-rotate in our [example](https://modelviewer.dev/examples/annotations/index.html#cameraViews), it correctly rotates about the given camera-target. Perhaps you can paste in a screen recording?

**Reply by bamiendev** [reply]
Thanks for your feedback. I have one more question below

**Comment by bamiendev** [comment]
https://github.com/google/model-viewer/assets/116520283/b60bcc76-14b9-41fc-806b-ad3ae1c83716

It looks something like this, the parameters set in the button slot's data-orbit and data-target are all misaligned

**Reply by bamiendev** [reply]
When I enable auto-rotate function

**Reply by elalish** [reply]
From your video I'd say everything looks perfectly aligned. The only strange UX part is that you have a controls box covering most of the bottom of the render, so the center point of your MV element doesn't look very visually centered anymore. You might consider putting the controls below or beside the MV element instead of on top. 

If I'm still missing something, you could make a mock image of how you expect it to look.
